Faq section aligned and styled in mobile view

harshb02 opened the pull request:
# Description
Faq section aligned and styled in mobile view

Please include a summary of the change and which issue is fixed.
Fixes #390 

## List any dependencies that are required for this change 
Na

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
On my local host

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
   - 
![WhatsApp Image 2021-04-19 at 8 52 14 PM](https://user-images.githubusercontent.com/58955446/115261702-79a1c580-a151-11eb-8581-df7d6662b9c8.jpeg)

    - After :
    
![WhatsApp Image 2021-04-19 at 8 51 29 PM](https://user-images.githubusercontent.com/58955446/115261732-7f97a680-a151-11eb-90e2-fc03bb3d7f20.jpeg)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

ravikr126 commented:
REMOVE MERGE CONFLICTS @harshb02

harshb02 commented:
Removed the conflicts @ravikr126

harikesh409 reviewed (commented):
Aren't these changes same as #429

harshb02 commented:
> Aren't these changes same as #429 

No in #429 I have aligned the Sharpen your Skills sectionin mobile view and in this PR I have aligned the FAQ section in mobile view.

harikesh409 commented on index.html:
```diff
@@ -146,6 +146,7 @@
 
     <!-- header ends -->
     <!--social media sidebar starts-->
+    <div class="d-none d-lg-block">
```
Correct the indentations.

harikesh409 commented:
> > Aren't these changes same as #429
> 
> No in #429 I have aligned the Sharpen your Skills sectionin mobile view and in this PR I have aligned the FAQ section in mobile view.

Okay both the changes are present here so I thought they were the same changes anyways do the requested changes we can merge it.

harshb02 commented:
I have removed the div because it was for not displaying the social sidebar in the mobile view but eventually it was done by someone else so I have removed the div and have done the changes @harikesh409

harikesh409 commented:
> I have removed the div because it was for not displaying the social sidebar in the mobile view but eventually it was done by someone else so I have removed the div and have done the changes @harikesh409

okay resolve the conflicts so that we can merge.

ravikr126 commented:
resolve the conflict @harshb02

harshb02 commented:
Removed the conflicts @ravikr126  @harikesh409
